doc/reference_doc_guidelines.md

```diff
@@ -188,23 +186,19 @@ examples:
                --ssh-key-values @key-file
   ```
 
-- Not all examples can be shell agnostic due to the quoting differences between Bash and PowerShell. For examples that only run in Bash, provide a second example for PowerShell. If you are testing examples in PowerShell, here are some considerations:
-  - [Quoting issues with PowerShell](https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md)
```

Comment:
Did you remove them because they are already at the beginning of this file?

Reply:
@jsntcy, yes